tools/datafix/reimport_gcs_record.py

```diff
@@ -150,6 +168,13 @@ def main() -> None:
       dest="tmpdir",
       default="/tmp",
       help="Local directory to copy to from GCS")
+  parser.add_argument(
+      "--bucket",
+      action="store",
+      dest="bucket",
+      default=None,
+      help=("Override the bucket name (and with a colon + path, the path) "
```

Comment:
Can you add an example here? The help text is a bit hard to parse. From looking at the source I think you mean 
```
bucket:optional/path/prefix/<individual-item-paths>
```

Reply:
I've expanded the help text in [a03a80e](https://github.com/google/osv.dev/pull/2462/commits/a03a80e5a7f2d40c6c17d8059def0824741dce5a)